cluster/charts/rook-ceph/templates/role.yaml

```diff
@@ -92,6 +92,8 @@ rules:
   - get
   - list
   - watch
+  - create
+  - update
```

Comment:
What if we change this to just allow creating and updating services? I'm not sure the mgr needs to create and update pods and pods/log also right?

Reply:
Also, it seems to me that it doesn't ever need to create the service, only update the one that the operator creates, right? And there should be no need to delete the service.

Reply:
This RBAC is mainly needed for the rook mgr module, which manages a number of resources. Really we should make most of this mgr RBAC optional in case the rook mgr module is enabled, but let's save that for a future day when it's clear how much effort will be put into the module.

Reply:
It's not that big of a deal, but my thinking is that the fewer permissions we give the mgr the better for security reasons.

Reply:
Agreed that fewer permissions are the goal, but currently the permissions are still required by the mgr. We just don't need them very often since the rook mgr module is usually disabled.